creator-node/sequelize/migrations/20200911004845-allow-track-and-audiusUsers-appends.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(`
+      BEGIN;
+      -- replace a table in place with extra trackBlockchainId column
+      CREATE TABLE "Files_new" ( like "Files" );
+      ALTER TABLE "Files_new" ADD COLUMN "trackBlockchainId" INTEGER;
+      INSERT INTO "Files_new" (SELECT f.*, t."blockchainId" FROM "Files" f LEFT OUTER JOIN "Tracks" t ON f."trackUUID" = t."trackUUID");
+      ALTER TABLE "Files_new" DROP COLUMN "trackUUID";
+      ALTER TABLE "Files" RENAME TO "Files_old";
+      ALTER TABLE "Files_new" RENAME TO "Files";
+      DROP TABLE "Files_old" CASCADE;
+
+      -- add back unique and pkey
+      ALTER TABLE "Files" ADD CONSTRAINT "Files_fileUUID_key" UNIQUE ("fileUUID");
```

Comment:
is this constraint not implicitly added by the below line?

Reply:
yes, youre right. we don't need this. i was just bringing the table to parity with what it was before

Reply:
👍